tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -916,3 +917,63 @@ def process(self, image, annotation_meta=None):
         image.metadata.update({'multi_infer': True, 'scales': scales})
 
         return image
+
+
+class Resize3D(Preprocessor):
```

Comment:
please move to resize.py file

Reply:
I'll move all brats-oriented preprocessors to brats_preprocessor.py